examples/using/using.ino

```diff
@@ -0,0 +1,31 @@
+/*
+  This example demonstrate pH value reading
+*/
+#include "Arduino.h"
+
+// Include library
+#include "TroykaPH.h"
+
+TroykaPH phMeter(A4); // set used analog pin
+
+uint32_t showingTime;
+constexpr uint32_t INTERVAL = 3000;
+
+void setup() {
+    phMeter.begin(); // if module not calibrated or...
+    // phMeter.begin(correction,zeroShift); // if you have it (use calibrate.ino) for it
+
+    Serial.begin(9600);
+    showingTime = millis() + INTERVAL; // show result once per 3 seconds
```

Comment:
No, I didn't mean it:) Use the classic millis() pattern:
```suggestion
    constexpr uint32_t interval = 3000;
    void setup() {
        ...
        showingTime = millis();
    }
    void loop() {
        ...
        if (millis() - showingTime >= interval) {
            showingTime = millis();
            ...
        }
    }
```

Reply:
Done with small corrections